ingest/sqs/ingester.go

```diff
@@ -104,5 +106,12 @@ func (i *sqsIngester) ProcessChangedBlockData(ctx sdk.Context, changedPools doma
 		return err
 	}
 
-	return i.sqsGRPCClient.PushData(ctx, uint64(ctx.BlockHeight()), pools, takerFeesMap)
+	for _, client := range i.sqsGRPCClients {
+		err = client.PushData(ctx, uint64(ctx.BlockHeight()), pools, takerFeesMap)
```

Comment:
May be some scope creep here, but could we thread this? And run each "PushData" in it's own thread? 

Happy to do this in a follow up PR tho, if we think it would work :thinking:

Reply:
Why didn't I think of that ?! Will do. Thanks man!

Reply:
Latest commit changed the push data logic in a separate thread